src/main/resources/org/jenkinsci/plugins/envinject/EnvInjectJobProperty/help-propertiesContent.html

```diff
@@ -1,7 +1,12 @@
 <div>
-    <p>
-        Give a set of key/value (KEY=VALUE).<br/>
-        All the properties name will be accessible as environment variables by their names.<br/>
-        You can use or override the properties specified in the above properties file.<br/>
-    </p>
+    <p>Give a set of key/value (one variable per line): <code>KEY=VALUE</code>.</p>
+
+    <p>All the properties name will be accessible as environment variables by their names. You can use or override the
+        properties specified in the above properties file.</p>
+
+    <p>These environment variables (<strong>only these</strong>) will be available in the Job. Use cases:</p>
+    <ul>
+        <li>Validate Subversion Repository with a URL like this: <code>http://server/svn/$REPO</code></li>
+        <li>Subversion SCM Polling</li>
+    </ul>
 </div>
```

Comment:
@oleg-nenashev I don't understand what you mean with **limitations**. Could you show an example the variable defined in this field *not resolvable*? Do you mean something like this `VAR1=WORKSPACE`?